nnpdf_data/theory.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    DAMPPOWERc: None #Power of the damping factor in FONLL for the c
+    DAMPPOWERb: None #Power of the damping factor in FONLL for the b
+    IC: int
+    IB: int
+    ModEv: str
+    ModSV: None#
+    XIR: float
+    XIF: float
+    NfFF: int
+    #MaxNfAs: int
```

Comment:
Both this and MaxNfPdf I believe are not actually used anymore in EKO (I remember a discussion in Amsterdam). Can you remind me? @giacomomagni @felixhekhorn

Reply:
both are still present in master [here](https://github.com/NNPDF/eko/blob/be97472bfe199fbf105d19d4b092aaa757488852/src/eko/io/runcards.py#L199), but are pending to be removed in https://github.com/NNPDF/eko/pull/295 (if ever we merge that PR :see_no_evil: )

Reply:
Ok so I'd say we remove them from here